README.md

```diff
@@ -1,12 +1,11 @@
 # WebView
 [![WebViewControl-Avalonia](https://img.shields.io/nuget/v/WebViewControl-Avalonia.svg?style=flat&label=WebView-Avalonia)](https://www.nuget.org/packages/WebViewControl-Avalonia/)
-[![WebViewControl-WPF](https://img.shields.io/nuget/v/WebViewControl-WPF.svg?style=flat&label=WebView-WPF)](https://www.nuget.org/packages/WebViewControl-WPF/)
 
-Avalonia/WPF control that wraps CefGlue webview control
+Avalonia control that wraps CefGlue webview control
 
-![Screenshot](./SampleWebView.Avalonia/screenshot.png)
+![Screenshot](./SampleWebView/screenshot.png)
 
-WebView lets you embed Chromium in .NET apps. It is a .NET wrapper control around [CefGlue](https://github.com/OutSystems/CefGlue) and provides a better and simple API. Likewise CefGlue it can be used from C# or any other CLR language and provides both Avalonia and WPF web browser control implementations. The Avalonia implementation runs on Windows and macOS. Linux is not supported yet.
+WebView lets you embed Chromium in .NET apps. It is a .NET wrapper control around [CefGlue](https://github.com/OutSystems/CefGlue) and provides a better and simple API. Likewise CefGlue it can be used from C# or any other CLR language and provides both Avalonia web browser control implementations. The Avalonia implementation runs on Windows and macOS. Linux is not supported yet.
```

Comment:
```suggestion
WebView lets you embed Chromium in .NET apps. It is a .NET wrapper control around [CefGlue](https://github.com/OutSystems/CefGlue) and provides a better and simple API. Likewise CefGlue it can be used from C# or any other CLR language and provides an Avalonia web browser control implementation. It runs on Windows and macOS. Linux is not supported yet.
```

Reply:
done